x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -15,301 +15,276 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
     key: 'windows.advanced.artifacts.global.manifest_relative_url',
     first_supported_version: '7.11',
-    documentation: '',
-  },
-  {
-    key: 'windows.advanced.artifacts.global.ca_cert',
-    first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Relative URL from which to download global artifact manifests. Default: /downloads/endpoint/manifest/artifacts-<version>.zip.',
   },
   {
     key: 'windows.advanced.artifacts.global.public_key',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: 'Public key used to verify the global artifact manifest signature.',
   },
   {
     key: 'windows.advanced.artifacts.global.interval',
     first_supported_version: '7.11',
-    documentation: '',
-  },
-  {
-    key: 'windows.advanced.artifacts.user.base_url',
-    first_supported_version: '7.11',
-    documentation: '',
-  },
-  {
-    key: 'windows.advanced.artifacts.user.ca_cert',
-    first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Interval between global artifact manifest download attempts, in seconds. Default: 3600.',
   },
   {
     key: 'windows.advanced.artifacts.user.public_key',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: 'Public key used to verify the user artifact manifest signature.',
   },
   {
     key: 'windows.advanced.artifacts.user.interval',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Interval between user artifact manifest download attempts, in seconds. Default: 3600.',
   },
   {
     key: 'windows.advanced.elasticsearch.delay',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: 'Delay for sending events to elasticsearch, in seconds. Default: 120.',
   },
   {
     key: 'windows.advanced.elasticsearch.tls.verify_peer',
     first_supported_version: '7.11',
-    documentation: 'default is true',
+    documentation: 'Whether to verify the certificates presented by the peer. Default: true.',
   },
   {
     key: 'windows.advanced.elasticsearch.tls.verify_hostname',
     first_supported_version: '7.11',
-    documentation: 'default is true',
+    documentation:
+      "Whether to verify the hostname of the peer is what's in the certificate. Default: true.",
   },
   {
     key: 'windows.advanced.elasticsearch.tls.ca_cert',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: 'Text blob of a certificate authority.',
   },
   {
     key: 'windows.advanced.malware.quarantine',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Whether quarantine should be enabled when malware prevention is enabled. Default: true.',
   },
   {
     key: 'windows.advanced.ransomware.mbr',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      "A value of 'false' overrides other config settings that would enable ransomware MBR. Default: true.",
```

Comment:
@magermark @gabriellandau please review these fields:
```
windows.advanced.ransomware.mbr
windows.advanced.ransomware.canary
```

I see that this code is currently ifdefed out:
https://github.com/elastic/endpoint-dev/blob/5f7127a8dbf4850c71b65b0cf42ac1359df9c000/Elastic/Plugins/Policy/Lib/plugins/Ransomware.cpp#L29-L85

but I'm assuming it'll be uncommented by the 7.11 release?

Reply:
Ransomware will only be released as a silent feature for 7.11, so only this function should be used: https://github.com/elastic/endpoint-dev/blob/5f7127a8dbf4850c71b65b0cf42ac1359df9c000/Elastic/Plugins/Policy/Lib/plugins/Ransomware.cpp#L91-L160 

I'm not sure how ransomware being released as silent-only should affect schema changes for 7.11, so I'll defer to @ferullo as to whether ransomware should only be defined in a silent / diagnostic policy context.

Reply:
I agree. Ransomware should be removed here for now.

Reply:
removed in ffa65db31caa5a5bca2ad19868587d115a8ffe04